src/shadowsocks_config.ts

```diff
@@ -338,3 +339,63 @@ export const SIP002_URI = {
     return `ss://${userInfo}@${uriHost}:${port.data}/${queryString}${hash}`;
   },
 };
+
+export interface OnlineConfig {
+  // URL endpoint to retrieve a Shadowsocks configuration.
+  readonly url: string;
```

Comment:
`url` is too ambiguous. Perhaps `configLocation` is better.

Reply:
Done in c325f47.